dist/last-campaign.js

```diff
@@ -157,6 +183,24 @@ function getCookie(name) {
     }
 }
 
+/**
+ * Update cookie expiration
+ *
+ * @param {string} name - name of cookie
+ * @param {date} expires - expiration date/time of cookie
+ * @param {object} options - object containing cookie options
+ * @private
+ */
+function updateExpiration(name, expires, options) {
+    var existingValue = getCookie(name);
+
+    if (existingValue) {
+        setCookie(name, existingValue, merge(options, {
```

Comment:
I'd recommend leaving a comment about why there is a specific method for updating the `expires` through assignment.